modules/influxdb/src/main/java/org/testcontainers/containers/InfluxDBContainer.java

```diff
@@ -0,0 +1,157 @@
+package org.testcontainers.containers;
+
+import org.influxdb.InfluxDB;
+import org.influxdb.InfluxDBFactory;
+import org.testcontainers.containers.traits.LinkableContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.containers.wait.strategy.WaitAllStrategy;
+
+import java.util.UUID;
+
+/**
+ * @link https://store.docker.com/images/influxdb
+ */
+public class InfluxDBContainer<SELF extends InfluxDBContainer<SELF>> extends GenericContainer<SELF>
+    implements LinkableContainer {
+
+    public static final String VERSION = "latest";
```

Comment:
could you please fix the version? We're trying to not use `latest` anymore to avoid unexpected test failures when new image is released (like it happened with MySQL 8 :D )